### PR TITLE
test(workflow): add basic model test

### PR DESCRIPTION
**Pull request by kkscilife**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

add basic model test

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: .github/workflows/daily_tests.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    if: ${{ always() && !cancelled() && contains(needs.*.result, 'failure') && (github.ref_name == 'develop' || github.ref_name == 'main') }}
+    needs: [HF_model]
+    runs-on: [test_machine]
+    steps:
+    - name: mask env
+      run: |
+        echo "::add-mask::${{env.WORKSPACE_PREFIX}}"
+    - name: notify
+      run: |
+        curl -X POST -H "Content-Type: application/json" -d '{"msg_type":"post","content":{"post":{"zh_cn":{"title":"Internlm GitHubAction Failed","content":[[{"tag":"text","text":""},{"tag":"a","text":"Please click here for details ","href":"https://github.com/pjlab-sys4nlp/train_internlm/actions/runs/'${GITHUB_RUN_ID}'"},{"tag":"at","user_id":"'${{ secrets.USER_ID }}'"}]]}}}}'  ${{ secrets.WEBHOOK_URL }}
```
Comment (del-zhenwu) [inline .github/workflows/daily_tests.yaml:44]: We may update the URL here.

File: .github/workflows/daily_tests.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+jobs:
+  HF_model:
+    runs-on: [t_cluster]
+    steps:
+    - name: mask env
+      run: |
+        echo "::add-mask::${{env.WORKSPACE_PREFIX}}"
+    - uses: actions/checkout@v3
+
+    - name: load_hf_model
```
Comment (del-zhenwu) [inline .github/workflows/daily_tests.yaml:21]: We may add timeout in this stage.

File: .github/workflows/daily_tests.yaml (hunk 1 above)
Comment (kkscilife) [reply .github/workflows/daily_tests.yaml:44]: We will use variable about repo here.

**Review by del-zhenwu — APPROVED** [review]

File: tests/test_hf_model.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+import torch
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+prompts = ["你好", "what's your name"]
+
+
+def assert_model(response):
+    assert len(response) != 0
+    assert "user" not in response
+    assert "bot" not in response
```
Comment (del-zhenwu) [inline tests/test_hf_model.py:11]: We may remove it since it's not stable sometimes.

Comment (kkscilife) [reply tests/test_hf_model.py]: Get!

File: tests/__init__.py
Comment (ZwwWayne) [inline tests/__init__.py:1]: This file seems to be unnecessary and should be deleted. Can be merged after deleted.

Comment (kkscilife) [reply tests/__init__.py:1]: Done!

**Pull request merged**